src/components/Session/AssetCardEffect.jsx

```diff
@@ -0,0 +1,72 @@
+/*
```

Comment:
This file is unecessary. Making a clone of AssetCard2 complicates the code for almost no benefits